nfc/include/nfc_t4t_lib.h

```diff
@@ -250,7 +255,7 @@ int nfc_t4t_response_pdu_send(const u8_t *pdu, size_t pdu_length);
 
 /** @brief Set an NFC parameter.
  *
- * Allows to set an NFC configuration parameter.
+ * Allows to set any parameter defined as available by HAL_NFC.
```

Comment:
Why is the description changed for this API? HAL_NFC is in the nrfx - I don't think this function allows to change any parameter in HAL_NFC

Reply:
Honestly  I do not know. This had to be changed previously by somebody

Reply:
Hmm, then we need to determine which description is more up to date and sync nrfxlib with original docs

Reply:
I updated the dox in https://github.com/NordicPlayground/nrfxlib/pull/84, so you'd just need to rebase. Though manifest in fw-nrfconnect-nrf (https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1534) doesn't point to the updated dox in nrfxlib, as other manifest PRs were blocking the update.